app/views/search/_sitetour_modal.html.erb

```diff
@@ -10,10 +10,10 @@
       </div>
       <div class="modal-footer">
         <a class="button secondary" id="closeInitialTourModal" data-dismiss="modal" href="#">Close</a>
-        <button class="button" data-bind="click: function(){ $root.ui.sitetour.startTour(); }">Start Tour <i class="fa fa-arrow-circle-right"></i></button>
+        <button class="button" data-bind="click: $root.ui.sitetour.startTour">Start Tour <i class="fa fa-arrow-circle-right"></i></button>
         <p class="pull-left">
           <label>
-            <input class="toggleHideTour" id="toggleHideTour" type="checkbox" data-bind="click: function(){ $root.ui.sitetour.toggleHideTour(); return true;}" />
```

Comment:
Deleting this code broke the 'Do Not Show Again' checkbox - it would never check.  The other code was setup to let it check.